packages/graphback-cli/src/commands/db.ts

```diff
@@ -1,5 +1,5 @@
 import chalk from 'chalk';
-import { DropCreateDatabaseAlways } from 'graphql-migrations';
+import { DropCreateDatabaseAlways } from 'graphql-migrations-bk';
```

Comment:
IMHO we do not need backward compatbility

Reply:
I don't plan to keep it. I didn't want to unnecessarily disrupt the CLI for what is a spike. I am focusing on runtime for now and so this was the easiest thing to do to not break things.